pandas/plotting/_matplotlib/core.py

```diff
@@ -1089,6 +1213,34 @@ def _make_plot(self):
         plot_colorbar = self.colormap or c_is_column
         cb = self.kwds.pop("colorbar", is_numeric_dtype(c_values) and plot_colorbar)
 
+        # move matplotlib.errorbar specific keywords that are not accepted by
+        # ax.scatter to an err_kwds dict for use in the call to ax.errorbar
+        ebarkeys = {
+            param
+            for param in ax.errorbar.__dict__["__signature__"].parameters
```

Comment:
I think `inspect.signature(ax.errorbar)` or related should work.

Reply:
Thanks for the tip.  inspect.signature works well